packages/ui/app/src/docs-context/DocsContextProvider.tsx

```diff
@@ -138,6 +151,9 @@ export const DocsContextProvider: React.FC<DocsContextProvider.Props> = ({
             resolvedUrlPath,
             scrollToPathListeners.registerListener,
             selectedSlug,
+            isSearchDialogOpen,
```

Comment:
Not a huge deal but I would just make a new context for the search dialog, this context was intended to be for docs content/navigation. For maintainability I generally prefer narrower contexts vs a single-catch all contexts.

Reply:
Gotcha. Will pull it into a separate context.